test/players.js

```diff
@@ -139,6 +140,23 @@ describe('GET /players/:id', () => {
             .get(`/players/${testPlayer.id}`)
             .expect(httpStatus.OK);
     });
+
+    it('should return player\'s junction table records', async () => {
+        const testPlayer = await models.player.findOne();
+
+        const res = await supertest(app)
+            .get(`/players/${testPlayer.id}`)
+            .expect(httpStatus.OK);
+
+        const recordsFromResponse = res.body.playerTeamSeason;
+        const recordsFromService = await models.playerTeamSeason.findAll({
+            where: {
+                playerId: testPlayer.id
+            }
+        });
+        assert.deepStrictEqual(recordsFromResponse.length, recordsFromService.length,
```

Comment:
please add an empty line here to separate the assertion logic and make the code more readable